managed/services/telemetry/telemetry.go

```diff
@@ -208,17 +208,22 @@ func (s *Service) processSendCh(ctx context.Context) {
 }
 
 func (s *Service) prepareReport(ctx context.Context) *pmmv1.ServerMetric {
+	dataSources := s.locateDataSources(s.config.telemetry)
```

Comment:
I guess it's not needed every time init this dataSources map, because config file cannot be changed in the runtime. 

I guess you can move initailization of this variable to Run() or New() method.

Reply:
Done.